src/main/java/org/folio/fqm/service/EntityTypeInitializationService.java

```diff
@@ -62,7 +70,16 @@ public EntityTypeInitializationService(
   // called as part of tenant install/upgrade (see FqmTenantService)
   public void initializeEntityTypes() throws IOException {
     log.info("Initializing entity types");
-
+    String centralTenantId = "${central_tenant_id}";
+    try {
+      String rawJson = ecsClient.get("consortia-configuration", Map.of("limit", String.valueOf(100)));
+      DocumentContext parsedJson = JsonPath.parse(rawJson);
+      centralTenantId = parsedJson.read("centralTenantId");
+      log.info("ECS central tenant ID: {}", centralTenantId);
+    } catch (Exception e) {
```

Comment:
Is there a better exception we can catch here? I worry about this potentially hiding bugs if something unexpected happens